railties/lib/rails/commands/test/test_command.rb

```diff
@@ -4,8 +4,10 @@
 module Rails
   module Command
     class TestCommand < Base # :nodoc:
-      def help
-        perform # Hand over help printing to minitest.
+      no_commands do
+        def help
+          perform # Hand over help printing to minitest.
+        end
```

Comment:
I was misunderstanding. 
Unlike `no_commands`,` remove_command` is also removed from the list of commands held inside thor. Therefore, if use `remove_command`, can not dispatch that command.
In this case, I do not want to display help in the command list, but want to make dispatch possible, so could not use `remove_command`.